File: src/main/java/me/andre111/d20server/message/game/PlayerList.java
package me.andre111.d20server.message.game;

import java.util.List;

import me.andre111.d20server.message.Message;
import me.andre111.d20server.model.entity.game.GamePlayer;

@SuppressWarnings("unused")
public class PlayerList extends Message {
	private List<GamePlayer> players;
	
	public PlayerList(List<GamePlayer> players) {
		this.players = players;
	}
}

File: core/common/message/account/player-list.js
import { Message } from '../message.js';
import { registerType } from '../../util/datautil.js';

export class PlayerList extends Message {
    players;

    constructor(players) {
        super();
        this.players = players;
    }

    getPlayers() {
        return this.players;
    }
}
registerType(PlayerList);
